helm-chart/templates/deployment-connector.yaml

```diff
@@ -29,25 +29,21 @@ spec:
             - containerPort: 9201
               name: connector-port
           env:
-            - name: PGPORT
+            - name: TS_PROM_DB_PORT
```

Comment:
how are these env variables read by the connector if they aren't in the args?

Reply:
we use a package that is set up to search ENV for vars starting with TS_PROM